src/test/scala/com/minosiants/pencil/SmtpServer.scala

```diff
@@ -40,43 +40,43 @@ final case class SmtpServer(
 
     stream.flatMap {
       case In(raw, Ehlo(_)) =>
-        Stream.eval_(state.tryUpdate(_ :+ raw)) ++
+        Stream.eval(state.tryUpdate(_ :+ raw)).drain ++
```

Comment:
`Stream.exec` without drain
assuming `state.tryUpdate` returns a `F[Unit]`

Reply:
It returns a boolean.